router/parse.js

```diff
@@ -16,77 +18,88 @@ export default function parse(uri) {
     items: []
   };
 
+  const parsers = [
+    ...whitelistRegex,
+    PANELS_REGEX
+  ];
```

Comment:
see https://github.com/UXtemple/panels/pull/10/files#r81768838 we can probably get away without this